odp_importer/msg.txt

```diff
@@ -0,0 +1,7 @@
+Hello,
+
+Your username is {username}.
+
+Go to {url} to reset.
+
+{email}
```

Comment:
It might be good to flesh this out a bit, something like:
```
You're receiving this message because you have an account ({username}) with OpenDataPhilly. We are launching a major upgrade to OpenDataPhilly, and in order to continue using the site, you will need to reset your password. You will only need to do this once. If you no longer wish to continue accessing OpenDataPhilly, you can ignore this message.

To reset your password, click this link: {url} .

To learn more about this upgrade and what it means for open data in Philadelphia, visit <link to press release>.

Thanks, and we hope you continue to contribute to Philadelphia's open data community!
```

@jbranigan is there a press release or something we can insert above?